src/js-api/handlers/ls.ts

```diff
@@ -0,0 +1,125 @@
+import { createBaseplateFetch, exitWithError } from "../../cli-utils";
+import {
+  EndpointGetCustomerWebAppsResBody,
+  EndpointGetEnvironmentsResBody,
+  EndpointGetMicrofrontendsResBody,
+  EndpointGetMyCustomerOrgResBody,
+} from "@baseplate-sdk/web-app";
+import { logTable, warning } from "../../cli-logger";
+
+export async function list(args: ListArgs) {
+  const baseplateFetch = createBaseplateFetch(args);
+  const { id: customerOrgId, orgKey } =
+    await baseplateFetch<EndpointGetMyCustomerOrgResBody>(`/api/orgs/me`);
+  const resourceLister = new ResourceLister(orgKey, customerOrgId);
+  await resourceLister.list(args);
+}
+
+export interface ListArgs {
+  baseplateToken: string;
+  resource: "mfe" | "webapp" | "env";
+}
+
+export class ResourceLister {
```

Comment:
Why classes for this? I would have written it with just functions within the file, and consider classes to be a backend-ish style of coding that isn't necessary in javascript. Do you prefer classes - if so, why?

Reply:
I find classes more manageable and organized in long run, rather than keep adding functions and passing same arguments we keep them in class properties as in object-oriented manner using typescript, but maybe that's because I am coming from Java background, to be hones don't know if classes are more expensive once it compiles to javascript or not.